src/app/app.functions/package.json

```diff
@@ -9,6 +9,7 @@
   "author": "",
   "license": "ISC",
   "dependencies": {
-    "node-hubspot-api": "^1.3.1"
+    "@hubspot/api-client": "^7.0.1",
+    "axios": "^0.27.2"
```

Comment:
Are we sure this works? From the slack thread, I was expecting the `node-hubspot-api` dep to be removed, but I am surprised to see axios and api-client added. 

Since the latter deps are already bundled with the project<sup>*</sup>, do you know if this affects the runtime behavior when you `require('axios')`? Without testing it, I would fear two possibilities:

1. This change is a noop, and `require('axios')` inside your fn returns the instance we already bundle with the project regardless of what's listed in package.json, or...
2. This conflicts with or overrides the existing modules already installed and bundled

Based on your testing, what are you seeing?


* <sub>I am not totally clear on how those modules are bundled (a custom lambda layer?) but at first glance it seems like this would cause confusion at best if the versions in package.json don't match what's in runtime</sub>

Reply:
I reached out via slack to a few teams internally and @banderson and I discussed and for now we are just going to remove all the dependencies

Reply:
The discussion was resolved with us deciding to add the dependency overrides to the `package.json` with a README that talks a bit around how a developer can add more NPM packages.